jvmpetrel/pom.xml

```diff
@@ -4,7 +4,7 @@
 
 	<groupId>storm.petrel</groupId>
 	<artifactId>storm-petrel</artifactId>
-	<version>0.9.0-SNAPSHOT</version>
```

Comment:
It looks like you're removing build settings for older Storm versions. Why do this?